dds_web/api/user.py

```diff
@@ -60,6 +66,88 @@ def jwt_token(username):
 ####################################################################################################
 
 
+class ConfirmInvite(flask_restful.Resource):
+    def get(self, token):
+        """ """
+
+        s = itsdangerous.URLSafeTimedSerializer(app.config.get("SECRET_KEY"))
+
+        try:
+            # Get email from token
+            email = s.loads(token, salt="email-confirm", max_age=10000)
```

Comment:
10000 seconds is a little short maybe? Maybe 24 hours, aka 86400 seconds?

Reply:
Assuming it's not the same person creating the invite that will confirm the invite, that person might not chech their email that quick...